python/mxnet/gluon/loss.py

```diff
@@ -325,21 +322,27 @@ class CTCLoss(Loss):
         This should be used as the fifth argument when calling this loss.
 
     Input shapes:
-        `data` is an activation tensor without softmax.
+        `data` is an activation tensor (i.e. before softmax).
         Its shape depends on `layout`. For `layout='TNC'`, this
         input has shape `(sequence_length, batch_size, alphabet_size)`
+        Note that the last dimension with index `alphabet_size-1` is reserved for special
+        blank character.
 
-        `label` is the label index matrix.
+        `label` is the label index matrix with zero-indexed labels.
         Its shape depends on `label_layout`. For `label_layout='TN'`, this
-        input has shape `(label_sequence_length, batch_size)`
-        When `label_lengths` is not specified, the first occurrence of `padding_mask`
+        input has shape `(label_sequence_length, batch_size)`. Padding mask of value ``-1``
```

Comment:
this is removed right?

Reply:
The parameter for padding mask is removed. The logic for padding mask value is not and should not be removed.